apps/web/src/pages/brand/tabs/v2/BrandingForm.tsx

```diff
@@ -0,0 +1,232 @@
+    },
+    onError: (e) => {
+      errorMessage('Failed to upload branding image: ' + e.message);
+    },
+  });
+
+  const { setValue, handleSubmit, control } = useForm<BrandFormValues>({
+    defaultValues: {
+      fontFamilyValue: organization?.branding?.fontFamily || 'inherit',
+      brandColorValue: organization?.branding?.color || DEFAULT_BRANDING_COLOR,
+      fontColorValue: organization?.branding?.fontColor || DEFAULT_BRANDING_COLOR,
+      brandLogoUrl: organization?.branding?.logo || '',
+      file: null,
+    },
+  });
+
+  const { isLoading, updateOrganizationBranding } = useUpdateOrganizationBranding({
+    onSuccess: () => {
+      successMessage('Branding info updated successfully');
+    },
+  });
+
+  const removeFile = () => {
+    setValue('file', null);
+    setValue('brandLogoUrl', '');
+  };
+
+  const handleUpload = async (files: File[]) => {
+    const file = files[0];
+    if (!file) return;
+
+    uploadToStorage({ file, type: UploadTypesEnum.BRANDING });
+  };
+
+  const saveBrandsForm = async ({
+    brandColorValue,
+    brandLogoUrl,
+    fontColorValue,
+    fontFamilyValue,
+  }: BrandFormValues) => {
+    const payload = {
+      brandColorValue,
+      brandLogoUrl: brandLogoUrl || undefined,
+      fontColorValue,
+      fontFamilyValue,
+    };
+
+    await updateOrganizationBranding(payload);
+  };
+
+  return (
+    <SettingsPageContainer title="Branding">
+      <LoadingOverlay visible={!organization}>
+        <form noValidate onSubmit={handleSubmit(saveBrandsForm)}>
+          <Stack gap={32}>
+            <Stack gap={10}>
+              <Text weight="bold">Brand logo</Text>
+              <Controller
+                render={({ field }) => (
+                  /**
+                   * group className is used to target the dropzone overlay styles
+                   * REF: https://panda-css.com/docs/concepts/conditional-styles#group-selectors
+                   */
+                  <div className={cx('group', dropzoneWrapperStyles)}>
+                    {field.value && (
+                      <div className={dropzoneOverlayStyles}>
+                        <div>
+                          <ActionButton onClick={() => dropzoneRef.current?.()} Icon={IconOutlineFileUpload} />
+                          <Text color={colors.B70}>Update</Text>
+                        </div>
+
+                        <div>
+                          <ActionButton onClick={removeFile} Icon={IconDelete} />
+                          <Text color={colors.B70}>Remove</Text>
+                        </div>
+                      </div>
+                    )}
+                    <Dropzone
+                      styles={{
+                        root: {
+                          background: 'none',
+                          border: 'none',
+                        },
+                      }}
+                      openRef={dropzoneRef}
+                      accept={Object.keys(MIME_TYPE_TO_FILE_EXTENSION)}
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** While this works for now, if we add another file type to that look-up that isn't valid in this situation, this will be a problem. Let's safeguard ourselves by picking the specific subset we want to accept‏